### DISCUSSION TITLE
How to cancel "--rm" in running `podman` container

### DISCUSSION THREAD

**Discussion by abitrolly** [containers/podman#16081] [Q&A] [answered]
I am using this helper script to run containerized program in current directory ([full source](https://github.com/yakshaveinc/linux/blob/master/bin/runin-podman.sh)).
```bash
#!/usr/bin/env bash

# name of current dir
NAME=$(basename "$PWD")

podman run -v "$(pwd):/root/$NAME":Z -w "/root/$NAME" -it "$@"
```

Now I want to add the flag `--rm` to avoid cleaning up containers once in a while. But there are cases when I want to stop container and then access it again later. Is there a way to prevent removal of a container that started with `podman run --rm`?

**Comment by Luap99** [comment]
No it is not possible to prevent removal when--rm if it is set.

**Reply by abitrolly** [reply]
Is it possible to shapshot the container somehow while it is running?

**Reply by Luap99** [reply]
podman container clone should work I guess

**Reply by abitrolly** [reply]
Works like a charm.
```
podman container clone mycontainer mycloned
podman start mycloned --attach
```
And `mycloned` is not cleaned up when it exits, which is what i need.

**Comment by abitrolly** [comment, marked as answer]
As suggested by @Luap99.
```
podman container clone mycontainer mycloned
podman start mycloned --attach
```
